pennylane_cirq/simulator_device.py

```diff
@@ -180,6 +187,11 @@ def analytic_probability(self, wires=None):
         probs = self._get_computational_basis_probs()
         return self.marginal_prob(probs, wires)
 
+    def _convert_to_density_matrix(self, state_vec):
+        """Convert ``state_vec`` into a density matrix."""
+        dim = 2**self.num_wires
+        return np.kron(state_vec, state_vec.conj()).reshape((dim, dim))
+
```

Comment:
This is moved, but the `MixedStateSimulatorDevice` inherits from the base `SimulatorDevice` so both classes have access to this helper function.